scripts/globals/items/high_breath_mantle.lua

```diff
@@ -0,0 +1,53 @@
+-----------------------------------------
+-- ID: 15487
+-- Item: High Breath Mantle
+-- Item Effect: HP+38 / Enmity+5
+-----------------------------------------
+
+require("scripts/globals/settings");
+require("scripts/globals/status");
+
+-----------------------------------------
+-- OnItemCheck
+-----------------------------------------
+
+function onItemCheck(target)
+	if (target:getStatusEffect(EFFECT_ENCHANTMENT) ~= nil) then
+		local effect = target:getStatusEffect(EFFECT_ENCHANTMENT);
```

Comment:
put this before the if, then just have the if check "if (effect ~= nil)", that way you don't iterate the whole buff list twice